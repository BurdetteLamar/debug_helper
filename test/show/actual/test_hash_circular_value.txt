---
Hash (size=2 name='test_hash_circular_value'):
  Pair 0:
    Key:
      Symbol (size=1 encoding=US-ASCII): :a
    Value:
      Hash (size=0): {}
  Pair 1:
    Key:
      Symbol (size=1 encoding=US-ASCII): :b
    Value:
      Hash (size=2):
        Pair 0:
          Key:
            Hash (size=0): {}
          Value: Fixnum 0
        Pair 1:
          Key: Hash {{}=>0, {...}=>0}
          Value: Fixnum 0
