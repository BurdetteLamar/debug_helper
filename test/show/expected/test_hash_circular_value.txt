---
Hash (size=2 default=nil name=test_hash_circular_value):
  Pair 0:
    Key:
      Symbol (size=1): :a
    Value:
      Hash (size=0 default=nil name=0): {}
  Pair 1:
    Key:
      Symbol (size=1): :b
    Value:
      Hash (size=2 default=nil name=1):
        Pair 0:
          Key:
            Hash (size=0 default=nil name=0): {}
          Value: 0 (Fixnum)
        Pair 1:
          Key: "{{}=>0, {...}=>0} (Hash)"
          Value: 0 (Fixnum)
